engine/src/main/java/com/datatorrent/stram/StramClient.java

```diff
@@ -125,6 +125,8 @@
   private String files;
   private LinkedHashSet<String> resources;
   private Set<String> tags = new HashSet<>();
+  private String customKeystoreConfig;
```

Comment:
Do these need to be stored? Can they not be fetched on demand from the configuration as there is only one place they are needed and the StramClient instances don't seem to be reused across multiple application launches from the cli.